### PR TITLE
Added tests for example scripts

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
Similar to some other repos, I have added tests for the example scripts. This will ensure that any changes in API will be reflected in these scripts, and perhaps will increase our testing coverage slightly.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

## Testing
I added new tests.

## Checklist
_Put an `x` in the boxes that apply._

- [x] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [x] I have run unit and regression tests which pass locally with my changes
- [x] I have added new tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

### CODE REVIEW

**Review by marcomangano — DISMISSED** [review]
Looks good, thanks for working on this!
I assume that accumulating the output files in the `self.output_file_list` attribute is necessary for testflo to use the `tearDown` method at the end of the run, right?

**Comment by nwu63** [comment]
> I assume that accumulating the output files in the `self.output_file_list` attribute is necessary for testflo to use the `tearDown` method at the end of the run, right?

Yeah this approach requires `self.output_file_list` to be set in the test method. Perhaps it would be a good idea to set this to an empty list in the `setUp` function?

**Review by marcomangano — APPROVED** [review]

**Comment by joanibal** [comment]
it looks like input_files/get-input-files.sh is an empty file?
why do we run an empty script? for inter repo consistency?

File: setup.py [hunk 1]
```diff
@@ -1,49 +1,21 @@
...
-        booktitle = {Proceedings of the 13th AIAA/ISSMO Multidisciplinary Analysis Optimization Conference},
-        year = {2010},
-        note = {AIAA 2010-9231},
-        month = {September},
-        address = {Fort Worth,~TX},
-        author = {Gaetan K. W. Kenway and Graeme J. Kennedy and Joaquim R. R. A. Martins}
-      }
-      
-      """,
+    long_description=long_description,
```
Comment (joanibal) [inline setup.py:18]: nice

**Review by joanibal — APPROVED** [review]

**Comment by joanibal** [comment]
Looks good, just one small question

**Comment by nwu63** [comment]
> it looks like input_files/get-input-files.sh is an empty file?
> why do we run an empty script? for inter repo consistency?

Nope the file exists, see [here](https://github.com/mdolab/pygeo/blob/master/input_files/get-input-files.sh). It's shown as an empty diff because the only change I made there was add the execution permission via `chmod +x`. `git` keeps track of (only) this bit of file permissions so when you pull the file it will be automatically set as an executable or not.

**Comment by joanibal** [comment]
Oh, I see. 
Thanks for explaining that

**Pull request merged**